Sources/PackageCollections/Utility.swift

```diff
@@ -30,11 +27,31 @@ struct NotFoundError: Error {
     }
 }
 
-// Model Extensions
+internal extension Result {
```

Comment:
this is used for compactMap situations. maybe there is nicer way to do this

Reply:
For the success case, there's `get`

Reply:
right, but its throwing some can change this to `try? self.get` which is verbose